backend/src/main/java/gov/cdc/usds/simplereport/db/model/Facility.java

```diff
@@ -45,6 +44,13 @@
       inverseJoinColumns = @JoinColumn(name = "device_specimen_type_id"))
   private Set<DeviceSpecimenType> configuredDeviceSpecimenTypes = new HashSet<>();
 
+  @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
Do you have any concerns about eager loading the devices for a facility? Are there any places we list facility names only in the app where there might be a performance hit if we load devices along with it?

Reply:
I am not too concerned, previously we were Eager loading the `facility_device_specimen_type`, which is arguably a bigger object since its the device and the specimen type. I am down to change this to Lazy once the slow migration from `facility_device_specimen_type` to `facility_device_type` is done.